src/commands/free.ts

```diff
@@ -0,0 +1,46 @@
+import { CommandType } from "../types";
+import { logger } from "../logger";
+import { SlashCommandBuilder, SlashCommandStringOption } from "@discordjs/builders";
+import { CommandInteraction, CacheType, UserResolvable } from "discord.js";
+
+
+const freeModule: CommandType = {
+    data: new SlashCommandBuilder()
+        .setName("free")
+        .setDescription("Free them all !!!")
+        .addUserOption(option =>
+            option
+                .setName('user')
+                .setDescription('The user you want to free')
+                .setRequired(true)
+        ),
+    execute: async (interaction: CommandInteraction<CacheType>) => {
+        try {
+            const member = interaction.options.getUser("user") as UserResolvable;
+
+            let memberDisplayName = (await interaction.guild?.members.fetch(member))?.displayName;
+
+            if (!memberDisplayName) {
+                return interaction.reply({
+                    content: 'Cannot put this member in jail. Not sure why',
```

Comment:
probably a good idea to change error message to `Cannot let this member out of jail` or something similar

Reply:
Done.